terraform/api-gtw.tf

```diff
@@ -185,32 +192,56 @@ resource "aws_api_gateway_integration" "reporting-incidents_incident-get_integra
   }
 }
 
-resource "aws_api_gateway_method_response" "reporting-incidents_incident-get-response_200" {
+resource "aws_api_gateway_method_response" "reporting-incidents_incident-mock-get-response_200" {
   rest_api_id = aws_api_gateway_rest_api.reporting-incidents.id
-  resource_id = aws_api_gateway_resource.reporting-incidents_incident-resource.id
-  http_method = aws_api_gateway_method.reporting-incidents_get-incident.http_method
+  resource_id = aws_api_gateway_resource.reporting-incidents_incident-mock-resource.id
+  http_method = aws_api_gateway_method.reporting-incidents_get-mock-incident.http_method
   status_code = "200"
 }
 
-data "template_file" "reporting-incidents_incident-get-response_template" {
+data "template_file" "reporting-incidents_incident-mock-get-response_template" {
   template = file("../testdata/testdata.json")
 }
 
-resource "aws_api_gateway_integration_response" "reporting-incidents_incident-get-response_integration_200" {
+resource "aws_api_gateway_integration_response" "reporting-incidents_incident-mock-get-response_integration_200" {
   rest_api_id = aws_api_gateway_rest_api.reporting-incidents.id
-  resource_id = aws_api_gateway_resource.reporting-incidents_incident-resource.id
-  http_method = aws_api_gateway_method.reporting-incidents_get-incident.http_method
-  status_code = aws_api_gateway_method_response.reporting-incidents_incident-get-response_200.status_code
+  resource_id = aws_api_gateway_resource.reporting-incidents_incident-mock-resource.id
+  http_method = aws_api_gateway_method.reporting-incidents_get-mock-incident.http_method
+  status_code = aws_api_gateway_method_response.reporting-incidents_incident-mock-get-response_200.status_code
 
   response_templates = {
-    "application/json" = data.template_file.reporting-incidents_incident-get-response_template.rendered
+    "application/json" = data.template_file.reporting-incidents_incident-mock-get-response_template.rendered
   }
 
   depends_on = [
-    aws_api_gateway_integration.reporting-incidents_incident-get_integration
+    aws_api_gateway_integration.reporting-incidents_incident-get_mock-integration
   ]
 }
 
+# Endpoint to return real incidents
+resource "aws_api_gateway_method" "reporting-incidents_get-incident" {
+  authorization = "NONE"
```

Comment:
❓ Shoudn't we have something here ? Don't want real data leaking by mistake while we dev no ?

Reply:
AUTH is explored currently. We definitely want only authorized users to be able to access the data, but we will add that later (when we know how to do it).
This will definitely have to arrive before we go live.